x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/AbstractSearchableSnapshotsTestCase.java

```diff
@@ -63,19 +101,27 @@ protected CacheService randomCacheService() {
         if (randomBoolean()) {
             cacheSettings.put(CacheService.SNAPSHOT_CACHE_RANGE_SIZE_SETTING.getKey(), randomCacheRangeSize());
         }
-        return new CacheService(AbstractSearchableSnapshotsTestCase::noOpCacheCleaner, cacheSettings.build());
+        if (randomBoolean()) {
+            cacheSettings.put(
+                CacheService.SNAPSHOT_CACHE_SYNC_INTERVAL_SETTING.getKey(),
+                TimeValue.timeValueSeconds(randomLongBetween(MIN_SNAPSHOT_CACHE_SYNC_INTERVAL.getSeconds(), Long.MAX_VALUE))
```

Comment:
I think we should bias towards smaller values, perhaps using `scaledRandomInt`?

Reply:
I pushed 8fe920be20ba2478f45e2940c688b9ab6ded559f to randomize between 1 and 120 seconds.